### ISSUE TITLE
mathjax support

### ISSUE THREAD

**Issue by 0-andy-0** [parpalak/upmath.me#23]
How do I get the equations rendered in the browser, I mean, using mathml/mathjax, instead of as images (forbidden by new accessibility legislation for public bodies)

**Comment by parpalak** [comment]
It's unlikely that MathJax will be supported. It has a different approach: https://github.com/mathjax/MathJax-src/issues/89.

Which accessibility features are missing in Upmath? The `img` tags have `alt` attributes with the source of the equation. Isn't that enough?
